### PR TITLE
Replace entrezGeneId with hugoGeneSymbol in fusion gene api

### PR DESCRIPTION
**Pull request by kalletlak**
And fix errors

### CODE REVIEW

File: src/pages/studyView/StudyViewUtils.spec.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ describe('StudyViewUtils', () => {
                     }]
                 }],
                 mutatedGenes: [{ "hugoGeneSymbols": ["GENE1"] }],
-                fusionGenes: [{ "entrezGeneIds": [1] }],
+                fusionGenes: [{ "hugoGeneSymbols": ["GENE1"] }],
```
Comment (zhx828) [inline src/pages/studyView/StudyViewUtils.spec.ts]: Should we use a different gene for fusion?

File: src/shared/api/generated/CBioPortalAPIInternal-docs.json [hunk 2]
```diff
@@ -1797,8 +1793,7 @@
       "required": [
         "counts",
         "entrezGeneId",
-        "hugoGeneSymbol",
-        "pValue"
```
Comment (zhx828) [inline src/shared/api/generated/CBioPortalAPIInternal-docs.json]: where is the pvalue?

Comment (kalletlak) [reply src/shared/api/generated/CBioPortalAPIInternal-docs.json]: not sure. I just updated the client.

File: src/pages/studyView/StudyViewUtils.spec.ts (hunk 1 above)
Comment (kalletlak) [reply src/pages/studyView/StudyViewUtils.spec.ts]: I think it's fine.

**Pull request merged**